koku/api/report/queries.py

```diff
@@ -233,15 +233,22 @@ def _build_custom_filter_list(self, filter_type, method, filter_list):
 
     def _check_for_operator_specific_filters(self, filter_collection, check_for_exclude=False):
         """Checks for operator specific fitlers, and adds them to the filter collection"""
```

Comment:
The logic in this function is getting really difficult to follow. Since this func is used to create 
```
composed_filters = self._check_for_operator_specific_filters(filters)
```
and
```
self.query_exclusions = self._check_for_operator_specific_filters(exclusions, True)
```
can we remove the exclusion stuff from this func and create a 
```
self.query_exclusions = self._check_for_operator_specific_exlusions
```
?

Reply:
Yeah! Originally we wanted to use the same functions, but I think this one as hit a point where we should separate them. Let me do that real quick and i'll ping you afterwards.